src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsidentifyremandperiodsapi/relevantremand/service/IdentifyRemandDecisionService.kt

```diff
@@ -46,6 +56,7 @@ class IdentifyRemandDecisionService(
         days = days,
         decisionByUsername = getCurrentAuthentication().principal,
         decisionByPrisonId = prisonerDetails.prisonId,
+        options = JacksonUtil.toJsonNode(objectMapper.writeValueAsString(options)),
```

Comment:
```suggestion
        options = objectMapper.valueToTree<JsonNode>(options),
```
not sure what's quicker here?